derivative/postgres_client_test.go

```diff
@@ -72,7 +72,7 @@ func TestPostgresAddPerson(t *testing.T) {
 	if err != nil {
 		panic(err)
 	}
-	assert.Equal(t, `{"name": "Barbara Liskov", "departments": [], "institutionalAffiliations": []}`, person)
+	assert.Equal(t, `{"name": "Barbara Liskov", "countries": null, "departments": [], "institutionalAffiliations": []}`, person)
```

Comment:
Can you explain why the org values are empty arrays and the countries value is `null`?

Reply:
This is debatable, but: It is asserting that there are no orgs, but the countries are not known.